test/TestAssets/TestProjects/DotnetRunTargetExtension/DotnetRunTargetExtension.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>$(CurrentTargetFramework)</TargetFramework>
+  </PropertyGroup>
+
+  <Target Name="_SetCustomRunArgs" BeforeTargets="ComputeRunArguments">
+    <PropertyGroup>
+      <RunArguments>$(RunArguments) extended</RunArguments>
+      <RunWorkingDirectory>$(MSBuildThisFileDirectory)</RunWorkingDirectory>
+    </PropertyGroup>
+  </Target>
```

Comment:
@fabiocav here's the simplest kind of extension in the new world - a NuGet package could contribute a target that `BeforeTargets="ComputeRunArguments"` and that's all that's required.

Reply:
👍🏼